yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
```

Comment:
There's a smaller number, but still significant ([1,578 according to taginfo](http://taginfo.openstreetmap.org/tags/natural=waterfall#overview)), of `natural=waterfall`. Are those worth including too?